src/app/static/src/js/components/runWorkflow/runWorkflowTable.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <table class="table-bordered">
+        <tr>
+            <th class="p-2">
+                Report
+            </th>
+            <th v-if="anyParams" class="p-2">
+                Parameters
+            </th>
+            <th class="p-2">
+                Status
+            </th>
+            <th class="p-2">
+                Logs
+            </th>
+        </tr>
+        <tr v-for="(report, index) in workflowRunStatus.reports" :key="report.key">
+            <td v-if="report.status === 'success'" class="p-2">
+                <a class="report-version-link" :href="reportVersionHref(report.name, report.version)">
+                    {{ report.name }}
+                </a>
+            </td>
+            <td v-else class="p-2">
+                {{ report.name }}
+            </td>
+            <td v-if="anyParams" class="p-2">
+                <run-workflow-parameters :report="workflowSummary.reports[index]"
```

Comment:
It looks like the `workflowSummary.reports` array and the `workflowRunStatus.reports` array do not neccessarilty have the reports in the same order! So just using a shared index here is not enough to pick out the correct report.

See screenshot here with the parameters flipped:
![Screenshot from 2022-03-16 15-29-34](https://user-images.githubusercontent.com/6736820/158627237-0acc3ed4-d6d5-48bc-9ca5-12a41094e38d.png)